### DISCUSSION TITLE
Which PSRAM chips are supported for rp2350?

### DISCUSSION THREAD

**Discussion by XanCraft21** [earlephilhower/arduino-pico#2452] [Q&A] [answered]
Hello, i was wondering which PSRAM chips are supported by the PSRAM library within this software package?

I have noticed that with the release of the new rp2350 there is now external ram support. I don’t think the pico version has a ram slot, but some of the upcoming adafruit boards do.

I have recently bought a few APS6404L chips and i’m not sure which platforms support them. I hope they can be used for more than just the PJRC Teensy 4.1.

Thank you for your help and answers.

**Comment by Macrosii** [comment, marked as answer]
Yes its support.  This is the chip used on the "Pimoroni Pico Plus 2" which was added to the boards menu in the latest release!
I also have a load of these I bought for Teensy 4.1 :)

**Reply by XanCraft21** [reply]
Thank you for answering my question, i’m glad the platform supports the ram chips i bought.
